pkg/trace/obfuscate/sql.go

```diff
@@ -209,7 +209,7 @@ func (f *groupingFilter) Reset() {
 // some elements such as comments and aliases and obfuscation attempts to hide sensitive information
 // in strings and numbers by redacting them.
 func (o *Obfuscator) ObfuscateSQLString(in string) (*ObfuscatedQuery, error) {
-	return o.ObfuscateSQLStringWithOptions(in, SQLOptions{QuantizeSQLTables: features.Has("quantize_sql_tables")})
+	return o.ObfuscateSQLStringWithOptions(in, SQLOptions{QuantizeSQL: features.Has("quantize_sql_tables") || features.Has("quantize_sql")})
```

Comment:
```suggestion
	return o.ObfuscateSQLStringWithOptions(in, SQLOptions{QuantizeSQL: features.Has("quantize_sql_tables") || features.Has("quantize_sql")})
```
Based on the above comment, we should also call the feature flag `"replace_sql_digits"` or `"quantize_sql_ids"`.

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8860/commits/83f3c03ef054618f7026747466d60e764694c041